README.md

```diff
@@ -16,22 +16,24 @@ The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the
 
 This is the flow that must be followed when releasing a new version of docs on openliberty.io. These steps update the `vNext`, `staging` and `draft` branches of the [docs repo](https://github.com/OpenLiberty/docs) so that the next version of docs can be worked on and ensures that fixes can still be made to the docs of the version being released.
 
+Before proceeding, make sure there are no current builds on the [prod pipeline](https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south). This is so merging pull requests later into prod does not impact any current deployments.
```

Comment:
I would be more explicit that you're referring to the "Build" step of the pipeline (active deployment are fine).  (We could technically note that auto-builds can be temporarily disabled and then re-enabled, but that's likely overkill)